usr/src/cmd/mdb/common/modules/zfs/zfs.c

```diff
@@ -444,7 +448,7 @@ blkptr(uintptr_t addr, uint_t flags, int argc, const mdb_arg_t *argv)
 	}
 
 	SNPRINTF_BLKPTR(mdb_snprintf, '\n', buf, sizeof (buf), bp, type,
-	    checksum, compress);
+	    checksum, 0, compress);
```

Comment:
1. don't we need to pass in the crypt_type?

2. dmu_ot[] isn't necessarily filled in here.  We need to read it from the target first.